EuclideanGeometry/Foundation/Axiom/Linear/Parallel.lean

```diff
@@ -82,9 +82,35 @@ section construction
 
 def inx_of_extn_line (r₁ r₂ : Ray P) (h : r₂.toProj ≠ r₁.toProj) : P := (cu r₁.toDir.toVec_nd r₂.toDir.toVec_nd (VEC r₁.source r₂.source) • r₁.toDir.toVec +ᵥ r₁.source)
 
-theorem inx_lies_on_fst_extn_line (r₁ r₂ : Ray P) (h : r₂.toProj ≠ r₁.toProj) : ((inx_of_extn_line r₁ r₂ h) ∈ r₁.carrier ∪ r₁.reverse.carrier) := sorry
+theorem inx_of_extn_line_symm (r₁ r₂ : Ray P) (h : r₂.toProj ≠ r₁.toProj) :
+  inx_of_extn_line r₁ r₂ h = inx_of_extn_line r₂ r₁ h.symm := by
+  have hsymm : cu r₁.toDir.toVec_nd r₂.toDir.toVec_nd (VEC r₁.source r₂.source) • r₁.toDir.toVec =
+      cu r₂.toDir.toVec_nd r₁.toDir.toVec_nd (VEC r₂.source r₁.source) • r₂.toDir.toVec + 
+      (r₂.source -ᵥ r₁.source) := by
+    have h1 : r₁.toDir.toVec_nd = r₁.toDir.toVec := rfl
+    have h2 : r₂.toDir.toVec_nd = r₂.toDir.toVec := rfl
+    have h := linear_combination_of_not_colinear_dir (VEC r₁.source r₂.source) h
+    nth_rw 1 [← EuclidGeom.cu_cv, Vec.mk_pt_pt] at h
+    rw [h1, h2, h, ← neg_vec r₁.source r₂.source, cu_neg, Complex.real_smul, neg_smul]
```

Comment:
since `h1 h2` are `rfl`, do you really need to rw h1 h2?  Maybe I am wrong.

Reply:
I have finished